feat: add `%_gnomeextensionsdir` (#265)
FyraLabs/devdocs: pages/terra/srpm.mdx

Used in [`%pkg_completions`](#pkg_completion), or manually when `%pkg_completions` cannot be used.

### %_gnomeextensionsdir

> Expands to: `%{_datadir}/gnome-shell/extensions/%{uuid}{:rpmspec}` or `%{_datadir}/gnome-shell/extensions{:rpmspec}`

Reviewer: Show what it expands to literally, since below it talks about the conditional

> **NOTE:** If the `%{uuid}{:rpmspec}` macro is defined, it will be appended to the path upon expansion: `%{_datadir}/gnome-shell/extensions/%{uuid}{:rpmspec}`.

Reviewer: ```suggestion
**NOTE:** %{uuid}{:rpmspec}` is usually defined in gnome-shell-extension packages. if the macro is defined in your spec file (`%global uuid PKG-UUID`), it will be appended to the path upon expansion as: `%{_datadir}/gnome-shell/extensions/%{uuid}{:rpmspec}`.
```

> Otherwise, it will expand without a `uuid`, which MUST be added manually: `%{_datadir}/gnome-shell/extensions{:rpmspec}`.

Reviewer: ```suggestion
Otherwise, it will expand without `%{uuid}`, as: `%{_datadir}/gnome-shell/extensions{:rpmspec}`.
Note that you SHOULD then need to add the extension's UUID after this macro.
```

Used in [`%pkg_completions`](#pkg_completion), or manually when `%pkg_completions` cannot be used.

### %_gnomeextensionsdir

> Expands to: `%{_datadir}/gnome-shell/extensions/%{uuid}{:rpmspec}` or `%{_datadir}/gnome-shell/extensions{:rpmspec}`

Author: as in this?
```suggestion
Expands to: `%{_datadir}/gnome-shell/extensions%{?uuid:/%{uuid}}{:rpmspec}`
```

Reviewer: Yeah
